model/src/main/java/org/esbtools/lightbluenotificationhook/NotificationEntity.java

```diff
@@ -73,25 +73,12 @@ public void setEntityVersion(String entityVersion) {
         this.entityVersion = entityVersion;
     }
 
-    @Transient
-    @Nullable
-    public String getEntityDataForField(String fieldPath) {
-        for (PathAndValue pathAndValue : entityData) {
-            if (Objects.equals(fieldPath, pathAndValue.getPath())) {
-                return pathAndValue.getValue();
-            }
-        }
-        throw new NoSuchElementException(fieldPath);
-    }
```

Comment:
This is useful in other projects (when writing Notification impls, it makes it easy to get the value of a particular field which you need to make appropriate DocumentEvents), I would like to keep this method here for convenience if you're okay with that.